grove/pyquil_utilities.py

```diff
@@ -0,0 +1,92 @@
+"""Module containing utilities that will eventually be implemented in pyquil."""
```

Comment:
The only thing in this module that is use in other parts of the code base is `prog_equality` which is a one liner. All the rest is only tested for its own sake. We can completely remove this file and just use the one-liner from `prog_equality` where-ever that is used in the the amplification tests.